src/config.ts

```
@@ -42,7 +41,6 @@ const defaultConfig: IConfig = {
     screenshotsDir: "./screenshots",
     threshold: 0.04,
     useDirect: false,
-    useGeckoDriver: false,
```

I like this.

I have a headless support PR coming in that silently ignores any and all non-gecko driver options so this is perfect.